codenames_backend/models/players.py

```diff
@@ -0,0 +1,18 @@
+from . import db, ma
+
+
+class Player(db.Model):
+    __tablename__ = "player"
```

Comment:
It will probably make sense to use the flask session id as the player id and then leave them unattached to teams or games.

We can then have a game_player bridge table to link the 2 and store the team at that point. It might also make sense to use a captain flag on the game_player table instead of storing red_team_captain and blue_team_captain explicitly in game. That way, the game_player can be used for most of the operations without needing to reference the game or player tables directly. I'm undecided on whether this last point is a good one or not though.

Reply:
Depends on a couple things. Are people connecting multiple times, or connecting after a disconnection? (Session ID then changes). Also makes it difficult to show a player all their games (although is probably irrelevant in this context). 

What operations are you thinking of that affect game_player, but not game? Almost all operations will modify game state...

Reply:
The other issue with storing captain on player (or some variant thereof) is it makes changing captain a more complicated operation, as several player records have to be changed, rather than a single field.